src/sources/hydra.rs

```diff
@@ -198,7 +192,6 @@ impl Worker {
                 stage.output.send(evt.into()).await.or_panic()?;
                 stage.ops_count.inc(1);
 
-                stage.chain_tip.set(point.slot_or_default() as i64);
```

Comment:
Maybe tx-indexer uses this, but what we have here is not a meaningful implementation anyway so this this should go